lstchain/scripts/lstchain_merge_hdf5_files.py

```diff
@@ -23,20 +23,15 @@
 from lstchain.io import auto_merge_h5files
 from lstchain.io import get_dataset_keys
 from lstchain.io.io import (
-    dl1_params_tel_mon_ped_key,
-    dl1_params_tel_mon_cal_key,
-    dl1_params_tel_mon_flat_key,
     dl1_mon_tel_catB_cal_key,
     dl1_mon_tel_catB_ped_key,
     dl1_mon_tel_catB_flat_key
 )
 
-default_keys_to_copy = [dl1_params_tel_mon_ped_key, 
-                       dl1_params_tel_mon_cal_key,
-                       dl1_params_tel_mon_flat_key,
-                       dl1_mon_tel_catB_cal_key,
-                       dl1_mon_tel_catB_ped_key,
-                       dl1_mon_tel_catB_flat_key
+default_keys_to_copy = [
+    dl1_mon_tel_catB_cal_key,
+    dl1_mon_tel_catB_ped_key,
+    dl1_mon_tel_catB_flat_key
```

Comment:
Are these safe from  having issues like those with the old monitoring tables? Do they have one entry per subrun?

Reply:
The new version has been tested, right? Changes look ok to me, but one never knows...

Reply:
> The new version has been tested on data and MC, right? Changes look ok to me, but one never knows...

Yes, merging is tested on both data and MC. I wanted to wait for #1257 to get the coverage reports back